modules/manage/pages/schema-reg/schema-reg-authorization.adoc

```diff
@@ -0,0 +1,385 @@
+= Schema Registry Authorization
+:page-categories: Management, Schema Registry, rpk, Security
+:page-aliases: manage:schema-reg-authorization.adoc
+:description: Learn how to set up and manage Schema Registry Authorization using ACL definitions that control user access to specific Schema Registry operations.
+// tag::single-source[]
```

Comment:
What's the related cloud-docs PR? We should connect the PRs in the playbook so we can verify the cloud version too?

Reply:
Not supported in Cloud